### ISSUE TITLE
Importing into VCarve Pro

### ISSUE THREAD

**Issue by TellefJ** [WikihouseUS/Alex-Wikished#1]
Just a note, importing the .skp file into VCarve Pro works as expected, but I found modifying the file in Sketchup 2017 then importing into VCarve Pro generates the error "Failed to open SketchUp file".  The workaround for this is to save the modified file as type SketchUp 2014, 2015, or 2016 (File, Save As, change the "Save as type" dropdown from its default setting).

Both my SketchUp Make 2017 (17.2.2555), and VCarve Pro - ShopBot Edition (8.517) were up-to-date when experiencing this behavior.

**Comment by SketchThis** [comment]
Yes, that's a known issue. In SketchUp you'll have to save it down to a 2015 file (File>Save As) and it'll work fine. It won't affect your SketchUp model in any way.
